Jenkinsfile

```diff
@@ -64,37 +64,118 @@ spec:
     }
   }
 
+  environment {
+    GOOGLE_APPLICATION_CREDENTIALS    = '/home/jenkins/dev/jenkins-deploy-dev-infra.json'
+  }
 
   stages {
```

Comment:
Total nit pick, but can we please get some code level documentation in for the different stages.  Thanks